stripe/http_client.py

```diff
@@ -144,7 +153,16 @@ def _handle_request_error(self, e):
 class UrlFetchClient(HTTPClient):
     name = 'urlfetch'
 
-    def __init__(self, verify_ssl_certs=True, deadline=55):
+    def __init__(self, verify_ssl_certs=True, proxies=None, deadline=55):
+
+        # no proxy support in urlfetch. for a patch, see:
+        # https://code.google.com/p/googleappengine/issues/detail?id=544
+        if proxies:
+            raise error.APIConnectionError(
```

Comment:
Great idea raising an error here! What do you think about making this a `ValueError` instead? I think it may be more appropriate given that this is an invalid option.